frontend/gist-frontend/src/Components/LoginOptionsComponent.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import UserLoginComponent from "./UserLoginComponent";
+import CreateUserComponent from "./CreateUserComponent";
+
+import { Box, Tab, Tabs, Heading } from "grommet";
+
+interface ILoginOptionsComponentProps {}
+
+interface ILoginOptionsComponentState {
+  isLoginTabActive: boolean;
+}
+
+export default class LoginOptionsComponent extends React.Component<
+  ILoginOptionsComponentProps,
+  ILoginOptionsComponentState
+> {
+  state = {
+    isLoginTabActive: true
+  };
+
+  onTabChange = () => {
+    this.setState({
+      isLoginTabActive: !this.state.isLoginTabActive
+    });
+  };
+
+  render() {
+    const { isLoginTabActive } = this.state;
+    const loginTitle: String = isLoginTabActive ? "login" : "create";
+    return (
+      <Box
```

Comment:
played around with a style if you wanna check it out

`<Box
        border={{
          color: "#1BD7BB",
          size: "large",
          style: "ridge",
          side: "all"
        }}
        round={true}
        width="large"
        height="large"
        background={{
          color: "#FFFFFF",
          opacity: "medium"
        }}
        margin={{
          right: "large",
          top: "large"
        }}
        basis="auto"
        elevation="xlarge"
        align="center"
        justify="center"
        overflow={{ vertical: "auto" }}
      >`